twoaxistracking/layout.py

```diff
@@ -121,6 +121,58 @@ def generate_field_layout(gcr, total_collector_area, min_tracker_spacing,
     relative_azimuth = np.mod(450-np.rad2deg(np.arctan2(Y, X)), 360)
     # Relative slope of collectors
     # positive means collector is higher than reference collector
-    relative_slope = -np.cos(np.deg2rad(slope_azimuth - relative_azimuth)) * slope_tilt  # noqa: E501
+    relative_slope = np.rad2deg(np.arctan(-np.cos(np.deg2rad(slope_azimuth - relative_azimuth))
+                                          * np.tan(np.deg2rad(slope_tilt))))
 
     return X, Y, Z, tracker_distance, relative_azimuth, relative_slope
+
+
+def get_max_shading_elevation(total_collector_geometry, tracker_distance,
```

Comment:
Would it make more sense to have this in `shading.py`?

Reply:
@kanderso-nrel